### PR TITLE
2.1 Basic Tests for Q, W, Shift+Q, Shift+W

### PR DESCRIPTION
**Pull request by ericfont**
Tests for basic functionally of going between 128th duration & whole duration with commands 'half-duration', 'double-duration', 'dec-duration-dotted', 'inc-duration-dotted'

Class Score gets methods cmdDoubleDuration(), cmdHalfDuration(), cmdIncDurationDotted(), cmdDecDurationDotted() which are public wrappers around private method cmdIncDecDurationDotted.

Simplified the text description of the commands.

### CODE REVIEW

**Comment by ericfont** [comment]
(this is 2.1-PR corresponding to #3104)

**Comment by lasconic** [comment]
It does't compile. MasterScore not defined.

**Comment by ericfont** [comment]
Oh sorry, I though I ran mtest, but I forgot to "build" tst_durationtype in qtcreator, hence I was running the old version.

Anyway, I've fixed the test file and it runs...so I guess wait and see if passes travis, and should be good to merge.

File: libmscore/score.h [hunk 1]
```diff
@@ -529,6 +529,10 @@ class Score : public QObject, public ScoreElement {
       void cmdAddOttava(Ottava::Type);
       void cmdAddStretch(qreal);
       void cmdResetNoteAndRestGroupings();
+      void cmdDoubleDuration()      { cmdIncDecDuration(-1, 0); }
+      void cmdHalfDuration()        { cmdIncDecDuration( 1, 0); }
+      void cmdIncDurationDotted()   { cmdIncDecDuration(-1, 1); }
+      void cmdDecDurationDotted()   { cmdIncDecDuration( 1, 1); }
```
Comment (Jojo-Schmitz) [inline libmscore/score.h]: Wouldn't it be better to use true and false here rather than 1 and 0?

Comment (ericfont) [inline libmscore/score.h]: Actually I'm thinking of eventually changing that parameter into a char or enum and be able to "step Double-Dotted" (Maybe Ctrl-Shift), "step Triple-dotted", and "step quad-dotted".

**Pull request merged**